src/udiskslinuxblock.c

```diff
@@ -2905,6 +2905,16 @@ build_command (const gchar *template,
   return command;
 }
 
+static gboolean
+need_partprobe_after_mkfs (const gchar *fs_type)
+{
+  /* udftools makes fake MBR since the 2.0 release */
+  if (g_strcmp0 (fs_type, "udf") == 0)
+    return TRUE;
+
+  return FALSE;
```

Comment:
Please add `inline` and make this just `return (g_strcmp0 (fs_type, "udf") == 0)`.

Reply:
Good point with inlining, we should make use of it more often! I wonder if it makes any real difference nowadays with all the compiler optimizations, giving a hint to a compiler never hurts though.